crates/blockifier/src/test_utils/contracts.rs

```diff
@@ -251,26 +248,28 @@ impl FeatureContract {
 
     pub fn get_compiled_path(&self) -> String {
         // ERC20 is a special case - not in the feature_contracts directory.
-        if let Self::ERC20(cairo_version) = self {
-            match cairo_version {
+        match self {
+            Self::ERC20(cairo_version) => match cairo_version {
                 CairoVersion::Cairo0 => ERC20_CAIRO0_CONTRACT_PATH,
                 CairoVersion::Cairo1 => ERC20_CAIRO1_CONTRACT_PATH,
+                CairoVersion::Native => {
+                    unreachable!("ERC20 contract is not supported in the native version.")
+                }
```

Comment:
Should we add a Native version then?

Reply:
I think it's out of scope for now, we didn't have this feature before